R/repository.R

```diff
@@ -44,8 +49,8 @@ git_find <- function(path = '.'){
 
 #' @export
 #' @rdname repository
-#' @param repo a path to an existing repository, or a `git_repository` object as
-#' returned by [git_open],  [git_init] or [git_clone].
+#' @param repo a path to an existing repository, or a git repository object as
```

Comment:
It seems like this should either be the actual class name (so, `git_repo_ptr`) and formatted as code or should be a more colloquial term (so, "git repository object") and not formatted as code.

Reply:
Which brings up: should this class have its own help topic?

Should its `print` method get split into a `format` and `print` method?

Should its `print` or `format` method identify the actual class? I find the ambiguity around `git_repo_ptr` vs `git repository` a bit confusing:

``` r
library(gert)
#> Linking to libgit2 v0.27.8, ssh support: YES, https support: YES
#> Default user: Jenny Bryan <jenny.f.bryan@gmail.com>

r <- git_init("~/tmp/ijk")

str(r)
#> Class 'git_repo_ptr' <externalptr>

r
#> <git repository>: /Users/jenny/tmp/ijk[@NA]
```

<sup>Created on 2020-05-25 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0.9001)</sup>